pkg/util/paramtable/component_param.go

```diff
@@ -2398,6 +2399,13 @@ func (p *queryCoordConfig) init(base *BaseTable) {
 	}
 	p.RetryNum.Init(base.mgr)
 
+	p.FileResourceMode = ParamItem{
```

Comment:
add comments, and what are the options?

Reply:
No, some times we need use different file resource mode for QueryCood and DataCoord.